Sources/Misc/Concurrency/OperationDispatcher.swift

```diff
@@ -56,15 +57,19 @@ class OperationDispatcher {
         Self.dispatchOnMainActor(block)
     }
 
-    func dispatchOnWorkerThread(delay: Delay = .none, block: @escaping @Sendable () -> Void) {
+    func dispatchOnWorkerThread(delay: JitterableDelay = .none, block: @escaping @Sendable () -> Void) {
         if delay.hasDelay {
             self.workerQueue.asyncAfter(deadline: .now() + delay.random(), execute: block)
         } else {
             self.workerQueue.async(execute: block)
         }
     }
 
-    func dispatchOnWorkerThread(delay: Delay = .none, block: @escaping @Sendable () async -> Void) {
+    func dispatchOnWorkerThread(after timeInterval: TimeInterval, block: @escaping @Sendable () -> Void) {
```

Comment:
This might be a small Josh nit but `delay` and `after` seem like very similar argument names and I would probably be confused by trying to figure out the difference between these 😅

Not a thing that _has_ to change in the PR but just a thought 🤷‍♂️

Reply:
I totally agree with you! The only difference is that `delay: JitterableDelay` doesn't let you specify the exact delay and includes a random jitter, while `after timeInterval: TimeInterval` allows you to specify the exact waiting time without a random jitter